ktlint-core/src/main/kotlin/com/pinterest/ktlint/core/KtLint.kt

```diff
@@ -177,58 +179,6 @@ object KtLint {
             .replaceFirst(UTF8_BOM, "")
     }
 
-    private fun userDataResolver(editorConfigPath: String?, debug: Boolean): (String?) -> Map<String, String> {
-        if (editorConfigPath != null) {
-            val userData = (
-                EditorConfigInternal.of(File(editorConfigPath).canonicalPath)
-                    ?.onlyIf({ debug }) { printEditorConfigChain(it) }
-                    ?: emptyMap<String, String>()
-                )
-            return fun (fileName: String?) = if (fileName != null) {
-                userData + ("file_path" to fileName)
-            } else {
-                emptyMap()
-            }
-        }
-        val workDir = File(".").canonicalPath
-        val workdirUserData = lazy {
-            (
-                EditorConfigInternal.of(workDir)
-                    ?.onlyIf({ debug }) { printEditorConfigChain(it) }
-                    ?: emptyMap<String, String>()
-                )
-        }
-        val editorConfig = EditorConfigInternal.cached()
-        val editorConfigSet = ConcurrentHashMap<Path, Boolean>()
-        return fun (fileName: String?): Map<String, String> {
-            if (fileName == null) {
-                return emptyMap()
-            }
-
-            if (fileName == STDIN_FILE) {
-                return workdirUserData.value
-            }
-            return (
-                editorConfig.of(Paths.get(fileName).parent)
-                    ?.onlyIf({ debug }) {
-                        printEditorConfigChain(it) {
-                            editorConfigSet.put(it.path, true) != true
-                        }
-                    }
-                    ?: emptyMap<String, String>()
-                ) + ("file_path" to fileName)
-        }
-    }
-
-    private fun printEditorConfigChain(ec: EditorConfigInternal, predicate: (EditorConfigInternal) -> Boolean = { true }) {
```

Comment:
ec4j does not provide any logging capabilities, so I just dropped this debug message.